include/benchmark/benchmark.h

```diff
@@ -1554,6 +1554,7 @@ class Fixture : public internal::Benchmark {
   BaseClass##_##Method##_Benchmark
 
 #define BENCHMARK_PRIVATE_DECLARE(n)                                 \
+  /* NOLINTNEXTLINE(misc-use-anonymous-namespace) */                 \
```

Comment:
Can this use `//` comment style, like elsewhere in the project?

Reply:
That was what I originally tried to use, but the answer is no. The preprocessor interprets the `\` at the end of the line as extending the comment to the next line. If you try to compile it, it fails with `error: multi-line comment [-Werror,-Wcomment]`.

The comment has to be scoped by using the `/* */` syntax, since the comment is embedded into the macro.

I also tried putting the comment before the macro, but that doesn't work, since the error occurs at the point of expansion.